src/common/Requirements.ts

```diff
@@ -144,16 +160,22 @@ export abstract class BaseSetup implements RequirementList {
                     testResult.tests.push({
                         duration: result.duration,
                         hasPassed: true,
-                        message: result.v,
-                        testResult: 'Passed'
+                        message: result.message,
+                        supplementalMessage:
+                            result.requirement.supplementalMessage,
+                        testResult: 'PASSED',
```

Comment:
now that we're printing the Passed/Failed messages to the output, we should probably put these words in message files so that they would eventually be translated.